### PR TITLE
Support codenotary verify/signing

### PR DESCRIPTION
**Pull request by pvizeli**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
+        docker \
+    && apk add --no-cache --virtual .build-dependencies \
+        build-base \
+        go \
+    \
+    && git clone -b v${VCN_VERSION} --depth 1 \
+        https://github.com/codenotary/vcn \
+    && cd vcn \
+    \
+    # Fix: https://github.com/codenotary/vcn/issues/131
```
Comment (ludeeus) [inline Dockerfile:26]: Should we wait for <https://github.com/codenotary/vcn/issues/131>?

File: Dockerfile [hunk 2]
```diff
@@ -1,40 +1,50 @@
...
+    && go get github.com/codenotary/immudb@4cf9e2ae06ac2e6ec98a60364c3de3eab5524757 \
```
Comment (ludeeus) [inline Dockerfile:27]: Should we store this SHA as a build arg?

File: builder.sh [hunk 3]
```diff
@@ -312,6 +323,9 @@ function run_build() {
             done
         done
     fi
+
+    # Singing image
+    codenotary_sign "${repository}/${image}:${version}"
```
Comment (ludeeus) [inline builder.sh:328]: We should probably limit this to when `"$DOCKER_PUSH" == "true"` there is no need to sign if we are not pushing the image.

File: Dockerfile (hunk 1 above)
Comment (frenck) [reply Dockerfile:26]: We have to update multiple places when the upstream issue gets fixed, I think this is fine for now.

File: Dockerfile (hunk 2 above)
Comment (frenck) [reply Dockerfile:27]: It's part of the workaround from above, it has no future, so would avoid.

File: builder.sh (hunk 3 above)
Comment (pvizeli) [reply builder.sh:328]: that is

Comment (pvizeli) [reply builder.sh:323]: image

**Review by frenck — APPROVED** [review]

File: builder.sh (hunk 3 above)
Comment (ludeeus) [reply builder.sh:323]: oh.. my bad :D

**Pull request merged**